packages/runtime/container-runtime/src/layerCompatState.ts

```diff
@@ -1,75 +0,0 @@
-/*!
```

Comment:
This file was renamed to `runtimeLayerCompatState.ts` and the data store state was added to it.